airbyte-integrations/connectors/source-fleetio/requirements.txt

```diff
@@ -0,0 +1,2 @@
+-e ../../bases/connector-acceptance-test
```

Comment:
Let's drop requirements.txt (delete the file) in favor or Poetry and `pyproject.toml`. I'm happy to help! Basically, we're going away from `setup,py` and `requirements.txt` and using Poetry for each connector. 

You can pretty much copy and paste this: https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-mailchimp/pyproject.toml — but make sure to NOT pin the CDK version. The new one is 0.65.something I believe.